### PR TITLE
Fix docstrings

### LINKED ISSUE #114: Revise / improve docs
Docs are currently a bit outdated (and probably full of typos / format issues, my bad on this >.<).

A compiled version of the docs can be found at https://talaia-labs.github.io/docs-teos/

Revision and update is required.

### PR DESCRIPTION
**Pull request by sr-gi**
This has been the most borring PR ever.

Fixes docstrings across the codebase.

Compiled version can be found at https://talaia-labs.github.io/docs-teos/teos.html

It'll be easier to check the compiled version for errors than the docstrings for the review.

close #114

### CODE REVIEW

File: common/exceptions.py [hunk 1]
```diff
@@ -16,30 +18,31 @@ def __str__(self):
         return message
 
     def to_json(self):
+        """Converts the exception to json format."""
         response = {"error": self.msg}
         response.update(self.kwargs)
         return response
```
Comment (bigspider) [inline common/exceptions.py:24]: Not really a part of this PR, but this could be a one-liner:
```python
    return {"error": self.msg, **kwargs}
```

Comment (sr-gi) [reply common/exceptions.py:24]: Will fix this in #216

File: contrib/client/help.py [hunk 2]
```diff
@@ -10,7 +10,7 @@ def show_usage():
         "\n\nGLOBAL OPTIONS:"
         "\n\t--apiconnect \tAPI server where to send the requests. Defaults to 'localhost' (modifiable in conf file)."
         "\n\t--apiport \tAPI port where to send the requests. Defaults to '9814' (modifiable in conf file)."
-        "\n\t-d, --debug \tShows debug information and stores it in teos_client.log."
+        "\n\t-d, --debug \tShows debug information and stores."
```
Comment (bigspider) [inline contrib/client/help.py:13]: Should remove ` and stores`.

**Review by bigspider — APPROVED** [review]
Some very minor comments, no big deal if left like this imho:
- I noticed that in some places you changed the docs for arguments or exceptions to always start Uppercase, but this is not done consistently across the codebase (actaully, in most places it's lowercase, I think).
- The docs for the `teos.lightning_server` module are empty, might be nicer to exclude them from the compiled logs, if possible.

**Comment by sr-gi** [comment]
> Some very minor comments, no big deal if left like this imho:
> 
> * I noticed that in some places you changed the docs for arguments or exceptions to always start Uppercase, but this is not done consistently across the codebase (actaully, in most places it's lowercase, I think).

Does it not follow this guidelines?

https://github.com/talaia-labs/python-teos/blob/1bbb863eba0b790788b16a8407f69db073942154/CONTRIBUTING.md#code-documentation

> * The docs for the `teos.lightning_server` module are empty, might be nicer to exclude them from the compiled logs, if possible.

Ups, that should not have been there, it's still WIP.

**Pull request merged**